Geometry/MTDNumberingBuilder/src/GeometricTimingDet.cc

```diff
@@ -90,10 +90,8 @@ GeometricTimingDet::GeometricTimingDet(DDFilteredView* fv, GeometricTimingEnumTy
   ddd_ = nav_type(nt.begin(), nt.end());
 }
 
-using namespace geant_units::operators;
-
 GeometricTimingDet::GeometricTimingDet(cms::DDFilteredView* fv, GeometricTimingEnumType type)
-    : trans_(fv->translation()),
+    : trans_(fv->translation() / dd4hep::mm),
```

Comment:
Has this line been wrong up to now? Before your change, the translations were in cm. Was that a bug?

Reply:
No, before it was redefined a right after the initialization list

Reply:
I see it now. I missed it before. It looks correct.

Reply:
@vargasa thanks for taking care of this, according to the unit test (and my check also on the GeometryBuilder) everything looks unchanged and consistent